packages/cursorless-engine/src/languages/LanguageDefinitions.ts

```diff
@@ -32,6 +32,11 @@ export interface LanguageDefinitions {
    */
   get(languageId: string): LanguageDefinition | undefined;
 
+  /**
+   * Clear the cache of all language definitions. This is run at the start of a command.
```

Comment:
worth saying why this is necessary. I'm also concerned that if you need to do this, then the cache is breaking things that don't happen as a result of a command, eg does hot reloading still work with the scope visualizer?

Reply:
I guess we blow away the language def every time it hot reloads so that's why it works?